app-text/zeal/zeal-0.7.1.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake edos2unix xdg-utils
+
+DESCRIPTION="Offline documentation browser inspired by Dash"
+HOMEPAGE="https://zealdocs.org/"
+SRC_URI="https://github.com/zealdocs/${PN}/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
can you change this to `GPL-3+`, see https://github.com/zealdocs/zeal/blob/main/src/app/main.cpp#L10-L12